### ISSUE TITLE
Add a quiet mode

### ISSUE THREAD

**Issue by frostyfrog** [matthieugrieger/mumbledj#109]
It would be nice if it were possible to mute all text (with and without announcing the next song) that is sent through mumble. Everyone on my server uses TTS and it's really annoying when the bot repeats (basically) what someone typed. Feature creep: Send a PM to admins what the announcing messages are.

**Comment by matthieugrieger** [comment]
Do you mean an option to disable all text messages that the bot sends? Because to me knowledge there isn't a way to disable TTS for all users.

**Comment by frostyfrog** [comment]
Yes, That's what I meant. I'm sorry if what I said didn't make much sense.
I was playing with a new toy (mumbledj) when I really should have been in
bed. :p Sorry about that.

~ Sent from my Alphabet phone

**Comment by MichaelOultram** [comment]
I think the only way to stop mumble reading out the message `!play https://www.youtube.com/watch?v=dQw4w9WgXcQ` is to right click the bot and send that as a private message. 

The bot would then still say `Rick Astley - Never Gonna Give You Up was added to the queue` but this could be adjusted (as well as any other message the bot sends). However, anything any other user sends to the global chat will be read out, and we have no control over that. 

Alternatively you can [disable text to speech for your own client](https://wiki.mumble.info/wiki/Mumbleguide/English#Text-To-Speech) instead, but this would remove all messages from being read.

**Comment by frostyfrog** [comment]
I actually started hacking away at the code a bit and I may have come up with something that may have worked out well. I'm still currently testing it out with a group of friends, so the code may change a bit. Once I have something that appears to work well, I'll fork and send a pull request to see what you think. :smiley:

**Comment by matthieugrieger** [comment]
Cool, looking forward to it. :)

**Comment by azlux** [comment]
Or just add your bot to a group where ACL don't allow to send message and ... that all !
